back/app/policies/application_policy.rb

```diff
@@ -12,8 +19,20 @@ def policy_for(record)
       Pundit.policy!(user_context, record)
     end
 
-    def scope_for(scope)
-      Pundit.policy_scope!(user_context, scope)
+    def scope_for(target_scope_or_klass)
+      (filtered_scope = handle_excluded_models(target_scope_or_klass)) and return filtered_scope if Rails.env.test?
```

Comment:
Maybe personal preference, but this might be clearer in my eyes
```suggestion
      if Rails.env.test?
         filtered_scope = handle_excluded_models(target_scope_or_klass)) 
         return filtered_scope if filtered_scope
      end
```

Reply:
I struggle still with some ruby shorthand. I may not even be reading this correctly!

Reply:
You are :-)

Reply:
I changed to the long form 👍